### ISSUE TITLE
Cannot use import from outside a module

### ISSUE THREAD

**Issue by breuerfelix** [babel/babel#11548]
## Bug Report

**Current Behavior**
Babel is not able to resolve the `import` statement as the native Browser.

Code setup:
```js
// index.js
import message from './message.js';
document.getElementById('output').innerHTML = message;
```
```js
// message.js
const message = "hello world";
export default message;
```

Working example without Babel:
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <div id="output"></div>
    <script src="index.js" type="module"></script>
  </body>
</html>
```

Example with Babel which is not working:
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <div id="output"></div>
    <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
    <script src="index.js" type="text/babel" data-type="module" data-presets=""></script>
  </body>
</html>
```

Babel throws the following error:
```
Cannot use import from outside a module
```
If i don't set the data-presets attribute, I get an error that require is not defined.

Am i missing something or is this a Bug ?

Why do i wanna get this going?  
If this will work you could set data-presets to "typescript" and call your typescript files "*.ts.js". Then you would be able to prototype Typescript and also split your code into different files without having to setup a bundler or ANYTHING.

To me, this is the sexiest and minimalist setup i could imagine.

**Comment by JLHwung** [comment]
In https://github.com/babel/babel/pull/11466 we support `data-type` annotation in `@babel/standalone`, it will be shipped in the next minor release 7.10.

Since there is an ongoing PR targeting to this issue, I am gonna close it. @breuerfelix Thanks for the report.

**Comment by breuerfelix** [comment]
@JLHwung hmm somehow i already read about that setting in another issue that it is possible :D

thanks alot for this info ! then i just have to wait some days, hope it will be soon since daaaamn a world without a bundler (even though a little slower) would be awesome :D
